src/app.py

```diff
@@ -17,8 +18,9 @@
 from requests.packages.urllib3.exceptions import InsecureRequestWarning
 import argparse
 from pathlib import Path
-from flask import Flask, g, jsonify, abort, request, json, Response
+from flask import Flask, g, jsonify, abort, request, json, Response, render_template
```

Comment:
@kburke this `render_template` is not being used

Reply:
Pushed change to remove, thanks!